modules/mobile/app/services/mobile/v2/appointments/proxy.rb

```diff
@@ -12,9 +12,11 @@ def initialize(user)
           @user = user
         end
 
-        def get_appointments(start_date:, end_date:, include_pending:, pagination_params:)
+        def get_appointments(start_date:, end_date:, include_pending:, pagination_params: {})
```

Comment:
Still leaving the param here in case we ever need to use it in the future. The VAOS service also defaults the value to an empty hash, so this doesn't impact functionality.